### ISSUE TITLE
Playwright: `TypeError: messages.result is not iterable`

### ISSUE THREAD

**Issue by JaapWeijland** [allynsweet/E2E-Mailbox#2]
When running the following with Playwright:

```
const foundEmail = await mailbox.waitForEmail(
    "Verifiëren e-mailadres " + emailAddress,
    100
);
```

Playwright crashes with the following error log:

```
TypeError: messages.result is not iterable

  31 |   await page.getByTestId("btn-register").click();
  32 |
> 33 |   const foundEmail = await mailbox.waitForEmail(
     |                      ^
  34 |     "Verifiëren e-mailadres " + emailAddress,
  35 |     100
  36 |   );

    at DeveloperMailService.fetchEmailList (/[...]/e2e/node_modules/e2e-mailbox/lib/services/developerMailService.js:98:40)
    at IntegrationMailbox.waitForEmail (/[...]/e2e/node_modules/e2e-mailbox/lib/index.js:106:28)
    at /[...]/e2e/tests/login.spec.ts:33:22
```

**Comment by allynsweet** [comment]
Fixed with #3, v1.1.4 thank you @JaapWeijland!

**Comment by JaapWeijland** [comment]
Awesome, my first published PR ever!
